lineman/app/components/inbox_page/inbox_page_controller.coffee

```diff
@@ -0,0 +1,16 @@
+angular.module('loomioApp').controller 'InboxPageController', ($rootScope, Records, CurrentUser, LoadingService, ThreadQueryService) ->
+  $rootScope.$broadcast('currentComponent', 'inboxPage')
```

Comment:
Hey so this whole controller is 100% duplicated at this point... there's nothing this controller can do at this point that DashboardPageController can't handle by itself.

I'm keeping it around because I think the designs will change to give the inbox some more custom functionality though.